cmd/minikube/cmd/config/validations.go

```diff
@@ -75,3 +76,11 @@ func IsValidPath(name string, path string) error {
 	}
 	return nil
 }
+
+func IsValidUrl(string, isoURL string) error {
```

Comment:
> What do you mean? 

@hferentschik I meant if we use "file://" in mac or in Linux that didn't work to get file downloaded using file URI. so as per parsing concern file:/, file:// and file:/// all are valid from ParseURI method.